README.md

```diff
@@ -466,8 +466,56 @@ Avoid including a key without a meaningful value:
 
 ### ✅
 
+The error object needs to have following: 
+ - Be consistent
+ - Have all required info
+ - Easy parsable
+ - Should be possible to build a solid UI on top, guiding the user what happened, and how to move on
+
+```
+{
+  "error": {
+    "localizedTitle": "Title goes here", // Optional title localized for end user
+    "localizedMessage": "Message goes here", // Optional message localized for end user
+    "message": "Invalid format, digits required", // Message for developer
+    "isRecoverable": true, // Is the error handled in the UI is fatal or can it be recovered, eg: try again
+    "code": "1202", // Code which the consumer of the API can parse and switch case on
```

Comment:
Should we just call this an identifier since it's a string and could basically also just be a string enum?

Reply:
If it can be a string, I'd agree with `identifier` and I think we should use a non-numeric string in the example as well.  Strings are more descriptive and you can use prefixes to "group" related errors.
@steffendsommer @Casperhr

Reply:
identifier and just a string, if you want to do it non-numeric in swift, fine. but lets not limit that.

FE is fine with whatever, as long as they can string compare